Mathlib/Data/Int/Order/Lemmas.lean

```diff
@@ -31,4 +31,13 @@ theorem natAbs_le_iff_mul_self_le {a b : ℤ} : a.natAbs ≤ b.natAbs ↔ a * a
   rw [← abs_le_iff_mul_self_le, abs_eq_natAbs, abs_eq_natAbs]
   exact Int.ofNat_le.symm
 
+@[simp]
+theorem isSquare_sign_iff {z : ℤ} : IsSquare z.sign ↔ 0 ≤ z := by
```

Comment:
`find_home` refuses to suggest anything for this besides whatever file I place it in, so maybe there is a better home.

Reply:
I could move it earlier at the cost of slightly tweaking the proof

Reply:
I guess that's fine.